app/boards/arm/bluemicro840/bluemicro840_v1.yaml

```diff
@@ -13,3 +13,6 @@ supported:
   - ieee802154
   - pwm
   - watchdog
+manufacturer: BlueMicro
+compatible: pro_micro
```

Comment:
I appreciate that having `compatible` as a string *or* array is prettier, it:
- doesn't immediately standout to users that it *can* be an array
- makes it slightly more complicated to process (most developers ignore the documentation, so the first array to appear may break stuff)

The list syntax avoids that complication.